core/scripts/chaincli/handler/report.go

```diff
@@ -20,7 +20,7 @@ import (
 	ocr2keepers "github.com/smartcontractkit/ocr2keepers/pkg"
 
 	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/keeper_registry_wrapper2_0"
-	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ocr2keeper/evm"
+	evm "github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ocr2keeper/evm20"
```

Comment:
Not sure if it will be easier to just replace all instances of 'evm' with 'evm20'? Would make it clearer in code that it only supports 2.0 version of evm